app/test_split_integer.py

```diff
@@ -2,20 +2,36 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    value = 10
+    number_of_parts = 3
+    parts = split_integer(value, number_of_parts)
+    assert sum(parts) == value
```

Comment:
Based on  clean code recommendations, will be better to do:
```suggestion
    assert split_integer(10, 3) == [3, 3, 4]
```
![image](https://github.com/mate-academy/py-split-integer/assets/87720583/1448aaa0-858f-4f33-87fc-30e9e5991cea)